package.json

```diff
@@ -223,7 +223,7 @@
     "lavamoat-browserify": "17.0.4",
     "lint-staged": "13.0.3",
     "mini-css-extract-plugin": "2.6.1",
-    "msw": "1.3.1",
+    "msw": "2.7.3",
```

Comment:
was seeing errors related to msw things bumping helped but required some refactoring